docs/ban-cooking.rst

```diff
@@ -12,11 +12,16 @@ Usage
 -----
 
 ::
-    ban-cooking <type>
+    ban-cooking <type|all|show>
```

Comment:
since `all` and `show` are literals and `<type>` can be repeated, this should be written as:
```
ban-cooking all [show]
ban-cooking <type> [<type> ...] [show]
```

`show` is something of an anomaly, though, since it wasn't documented before, was intended for debugging, and doesn't follow the pattern of other kinds of commands that do this kind of thing. We don't need to be faithful to the `show` syntax since it was never documented. I think it should be represented as a flag (`-v`, `--verbose`), or it can just remain undocumented.